.env.ci

```diff
@@ -43,6 +43,7 @@ REDIS_PORT=$REDIS_PORT
 REDIS_HOST=$REDIS_HOST
 
 # IPFS
+IPFS_CLUSTER_ROOT=http://localhost:8080
```

Comment:
@JGiter why is `IPFS_CLUSTER_ROOT` added to `.env` files but `IPFS_CLUSTER_PASSWORD` is not?

Reply:
I didn't added because local IPFS isn't authorized

Reply:
Probably it is still worth adding as a hint